Directory.Packages.props

```diff
@@ -28,4 +27,7 @@
     <PackageVersion Include="JsonSchema.Net" Version="7.2.3" />
     <PackageVersion Include="FSharp.Core" Version="8.0.401" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <GlobalPackageReference Include="Nerdbank.GitVersioning" Version="3.6.146" />
```

Comment:
I would probably only scope this to projects under the src/ folder

Reply:
That's an option. But why bother to add nuance?